### ISSUE TITLE
npm package

### ISSUE THREAD

**Issue by pickfire** [aep/pug-rs#2]
Is it possible to have a webpack loader using `pug-rs`?

**Comment by aep** [comment]
yeah. create a file pug_loader.js and copy the code from the README.md.

let me know if there's anything unclear with that.

**Comment by pickfire** [comment]
Should I use [wasm-pack](https://github.com/rustwasm/wasm-pack) for this?

**Comment by aep** [comment]
no not at all. just copy the code from the README into your project.
https://github.com/aep/pug-rs#with-webpack

**Comment by pickfire** [comment]
No, I don't mean for my project. I saw projects that do not need to modify their webpack config but just install a plugin to use it such as https://github.com/pugjs/pug-loader. I was just wondering if there could be such option so that people can just add this as a dependency and just use it without modifying anything.

**Comment by aep** [comment]
that would require publishing to npm. I have no plans on doing that, but contributions welcome :)

**Comment by pickfire** [comment]
@aep If I were to help to create the NPM package. I have some parts not clear.

- Package it as wasm for nodejs?
- Put it in this git repo or separate?

**Comment by aep** [comment]
btw zz creates npm packages automatically.
i'll probably rewrite this in zz anyway when i get around to doing webstuff
